rottenOranges-TLE.py

```diff
@@ -0,0 +1,36 @@
+class Solution(object):
+    def orangesRotting(self, grid):
+        """
+        :type grid: List[List[int]]
+        :rtype: int
+        """
+        fresh = 0
+        q =[]
+        m = len(grid)
+        n = len(grid[0])
+        for i in range(m):
+            for j in range(n):
+                if grid[i][j] == 2:
+                    q.append((i,j))
+                if grid[i][j] == 1:
+                    fresh += 1
+        dirs = ((0,1),(1,0),(-1,0),(0,-1))
+        count = 0
+        while q != None:

```

Comment:
while len(q)>0

Reply:
Yes. Just understood that while q != 0 leads to an infinite loop. It works when I change to len(q) > 0